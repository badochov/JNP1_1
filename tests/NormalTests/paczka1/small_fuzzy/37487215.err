Error in line 12: w2dz S293 344,8
