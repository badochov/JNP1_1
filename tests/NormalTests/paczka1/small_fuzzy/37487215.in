ZdWqzoehap S148 20,7
ZdWqzoehap S148 95,0
p5T S148 24,3
NndOH S922 82,6
q7v9jIC9 S148 12,8
NndOH S922 319,7
w2dz S293 67,8
q7v9jIC9 S148 41,2
aFwCkIvj S293 65,1
?w2dz
w2dz S293 74,8
w2dz S293 344,8
?S922
aFwCkIvj S293 260,4
w2dz S148 57,7
ZdWqzoehap S293 51,0