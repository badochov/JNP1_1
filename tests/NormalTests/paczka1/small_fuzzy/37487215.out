S922 237,1
